src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -1,5 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fields>
+        <fullName>InstallmentOppStageName__c</fullName>
+        <description>The Opportunity Stage that NPSP applies to each new installment Opportunity when you create a Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The Opportunity Stage that NPSP applies to each new installment Opportunity when you create a Recurring Donation.</inlineHelpText>
+        <label>Stage Name for Installment Opportunities</label>
+        <length>50</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>IsEnhancementRDEnabled__c</fullName>
```

Comment:
@meena-sfdc and @RadSFDeveloper I've been thinking about this field api name more, and after conferring with Mary and Joy, I'd like to go with field api name of `IsRecurringDonations2Enabled__c`. The label can remain to what Sam has approved. The two do not need to be in sync. From a system level this is "Recurring Donations 2.0" though we're communicating it as "Enhanced Recurring Donations". There may come a time when we eventually take RD another step further (RD3 for example) or even have another level of "enhanced". Using the "RecurringDonations2" in the api name allows us the option of changing the marketing label at any point while still ensuring a consistent use of the Api Name in the code (for example, the FeatureParameter for this is `IsEnabled_RecurringDonations2`).
CC @sam-knox @npsp-reedestockton

Reply:
Thanks for the confirmation, @force2b . The changes have been made and ready for review.